RELEASES.md

```diff
@@ -11,6 +11,13 @@
 ### Usability
 
 - Implemented `Eq` for `Timing` and `InputMap`.
+- `UserInput` is now generic over the maximum size of inputs for a `Chord` (the default is still `8`).

```

Comment:
IMO we should probably lower the default to something much smaller, like 4, now that it can be configured, to save space for the 99% of use cases that don't want more.